### PR TITLE
Add one user views in hr page

### PR DESCRIPTION
**Pull request by MS151994**
Add one user views in hr page  1/2

### CODE REVIEW

File: src/components/Dashboard/OneUser/OneUser.styles.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  & > svg {
+    margin: 5px;
+    cursor: pointer;
+    ${({ isOpen }) =>
+       isOpen &&
+       `
+    transform: rotate(-180deg)
+  `}
+  }
+}
```
Comment (Fremen1990) [inline src/components/Dashboard/OneUser/OneUser.styles.ts]: 👍

File: src/components/Dashboard/OneUser/OneUser.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+         </div>
+         <ButtonsBox isOpen={isOpen}>
+            <Button buttonTitle={'pokaż CV'} />
+            <Button buttonTitle={'brak zainteresowania'} />
+            <Button buttonTitle={'zatrudiony'} />
+            <AiOutlineCaretDown onClick={() => setIsOpen(!isOpen)} />
+         </ButtonsBox>
+      </UserContainer>
+   )
+}
```
Comment (Fremen1990) [inline src/components/Dashboard/OneUser/OneUser.tsx]: 👍

File: src/pages/Admin.page.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { PageContainer } from '../constants/Layout/Container.styles'
 import { Header } from '../components/Header/Header'
+import { OneUser } from '../components/Dashboard/OneUser/OneUser'
```
Comment (Fremen1990) [inline src/pages/Admin.page.tsx]: 👍

File: src/pages/Admin.page.tsx [hunk 4]
```diff
@@ -11,6 +12,7 @@ export const AdminPage = () => {
             <p>
                For example how to use page container with header component :)
             </p>
+            <OneUser />
```
Comment (Fremen1990) [inline src/pages/Admin.page.tsx]: 👍

**Review by Fremen1990 — APPROVED** [review]
All looks good 👍

**Review by Fremen1990 — DISMISSED** [review]

**Comment by MS151994** [comment]
desktop
mobile
tablet

File: src/components/Dashboard/OneUser/OneUser.tsx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+               <InterViewMeBox
+                  ratingBox={false}
+                  boxTitle={descriptions.practice}
+                  point={'tak'}
+               />
+               <InterViewMeBox
+                  ratingBox={false}
+                  boxTitle={descriptions.experience}
+                  point={'6 mies'}
+               />
```
Comment (Fremen1990) [inline src/components/Dashboard/OneUser/OneUser.tsx:101]: in future we can map through and use one component :)

**Review by Fremen1990 — APPROVED** [review]

File: src/components/Dashboard/OneUser/OneUser.tsx (hunk 5 above)
Comment (Fremen1990) [reply src/components/Dashboard/OneUser/OneUser.tsx:101]: Such a bautiful work 🥇

**Pull request merged**